### PR TITLE
Add ability to run CI test C96_atm3DVar.yaml to Gaea-C5

### LINKED ISSUE #2766: Setup cycling test C96_atm3DVar on Gaea-C5
### What new functionality do you need?

Add the ability to run CI test C96_atm3DVar and C96_atm3DVar_extended on Gaea-C5

### What are the requirements for the new functionality?

Clone, build, and install obsproc, prepobs, and fit2obs
Run C96_atm3DVar and C96_atm3DVar_extended CI cases

### Acceptance Criteria

Successful completion of C96_atm3DVar and C96_atm3DVar_extended CI test on Gaea-C5

### Suggest a solution (optional)

_No response_

### PR DESCRIPTION
**Pull request by DavidBurrows-NCO**
# Description

What: update files to run global-workflow C96_atm3DVar.yaml CI test.
Why: Refs #2766 

  Resolves #2766 
  Refs NOAA-EMC/prepobs#32
  Refs NOAA-EMC/Fit2Obs#28

# Type of change
- New feature (adds functionality)

# Change characteristics
- Is this a breaking change (a change in existing functionality)? NO
- Does this change require a documentation update? NO
- Does this change require an update to any of the following submodules? YES/NO (If YES, please add a link to any PRs that are pending.)

NOAA-EMC/prepobs [#35](https://github.com/NOAA-EMC/prepobs/pull/35)
NOAA-EMC/Fit2Obs [#29](https://github.com/NOAA-EMC/Fit2Obs/pull/29)

# How has this been tested?
Clone, build, and run CI test C96_atm3DVar successfully on Gaea-C5.

# Checklist
- [x] Any dependent changes have been merged and published
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] My changes generate no new warnings
- [ ] New and existing tests pass with my changes
- [ ] I have made corresponding changes to the documentation if necessary

### CODE REVIEW

File: env/GAEA.env [hunk 1]
```diff
@@ -34,12 +34,42 @@ else
...
+    ntasks_cycle=${ntiles:-6}
+    export APRUN_CYCLE="${launcher} -n ${ntasks_cycle} --cpus-per-task=${NTHREADS_CYCLE}"
+
+    export NTHREADS_GAUSFCANL=1
+    ntasks_gausfcanl=${ntasks_gausfcanl:-1}
+    export APRUN_GAUSFCANL="${launcher} -n ${ntasks_gausfcanl} --cpus-per-task=${NTHREADS_GAUSFCANL}"
+
+elif [[ "${step}" = "sfcanl" ]]; then
+
+    export NTHREADS_CYCLE=${threads_per_task:-14}
```
Comment (aerorahul) [inline env/GAEA.env:71]: `14` as default seems very particular.  Just curious.

**Review by aerorahul — DISMISSED** [review]
looks good.  just one comment.

File: env/GAEA.env (hunk 1 above)
Comment (DavidBurrows-NCO) [reply env/GAEA.env:71]: @aerorahul Just checked...all of the systems use 14 as default.

**Review by aerorahul — APPROVED** [review]

File: modulefiles/module_base.gaea.lua [hunk 2]
```diff
@@ -36,4 +38,13 @@ load(pathJoin("py-xarray", (os.getenv("py_xarray_ver") or "None")))
 setenv("WGRIB2","wgrib2")
 setenv("UTILROOT",(os.getenv("prod_util_ROOT") or "None"))
 
+--prepend_path("MODULEPATH", pathJoin("/gpfs/f5/ufs-ard/world-shared/global/glopara/data/git/prepobs/v" .. (os.getenv("prepobs_run_ver") or "None"), "modulefiles"))
+--load(pathJoin("prepobs", (os.getenv("prepobs_run_ver") or "None")))
+prepend_path("MODULEPATH", pathJoin("/gpfs/f5/ufs-ard/world-shared/global/glopara/data/git/prepobs/v1.1.0", "modulefiles"))
+load(pathJoin("prepobs", "1.1.0"))
```
Comment (DavidBurrows-NCO) [inline modulefiles/module_base.gaea.lua:44]: @aerorahul The new prepobs version Kate had me install is v1.1.0. The other systems are still using v1.0.2. Do you have a suggestion where to put prepobs_run_ver? Clearly not hardcoded like this...

Comment (DavidBurrows-NCO) [reply modulefiles/module_base.gaea.lua:44]: I see it Refs #2840.

Comment (aerorahul) [reply modulefiles/module_base.gaea.lua:44]: can I merge this PR?

Comment (DavidBurrows-NCO) [reply modulefiles/module_base.gaea.lua:44]: @aerorahul If you are okay with hard coded prepobs version for now until other systems update to v1.1.0?

Comment (aerorahul) [reply modulefiles/module_base.gaea.lua:44]: fine with me for now. @KateFriedman-NOAA any preference?

Comment (KateFriedman-NOAA) [reply modulefiles/module_base.gaea.lua:44]: Yeah that's fine. I will turn that hardcoded value back into the variable in the PR I will make soon to update `develop` to `obsproc/v1.2.0` and `prepobs/v1.1.0`.

Note to self in issue #2840.

**Review by WalterKolczynski-NOAA — APPROVED** [review]

File: modulefiles/module_base.gaea.lua (hunk 2 above)
Comment (DavidBurrows-NCO) [reply modulefiles/module_base.gaea.lua:44]: Thanks @KateFriedman-NOAA. @aerorahul I just updated my branch so should be ready to merge. Thanks.

**Review by KateFriedman-NOAA — APPROVED** [review]

**Comment by DavidBurrows-NCO** [comment]
Thanks @KateFriedman-NOAA @WalterKolczynski-NOAA @aerorahul

**Pull request merged**